test/e2e/network/netpol/network_policy.go

```diff
@@ -1249,6 +1252,67 @@ var _ = common.SIGDescribe("Netpol [Feature:SCTPConnectivity][LinuxOnly][Disrupt
 	})
 })
 
+var _ = common.SIGDescribe("Netpol [Feature:NetworkPolicyEndPort][LinuxOnly]", func() {
```

Comment:
hi ! thanks for this ~ is there a reason this is a new SIGDescribe? If so just comment on why ! :) THANKS for helping us to maintain the netpol suite ! :) otherwise i think these tests look good

Reply:
The reason is that I used to consider using skipper to consider whether to run it. But now we should remove skipper.(as @spiffxp say https://github.com/kubernetes/kubernetes/pull/99777#discussion_r638138521). So you think it is better to put it under the SIGDescribe `Netpol [LinuxOnly]` ?

Reply:
@jayunit100  I put it under the  SIGDescribe `Netpol [LinuxOnly]`. IF you feel no suitable, plz point out.